src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/SetActivationDeploymentsRule.cs

```diff
@@ -10,7 +10,7 @@ namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
     /// Check that the previous block hash is correct.
     /// </summary>  
     [PartialValidationRule]
-    public class BlockHeaderRule : ConsensusRule
+    public class SetActivationDeploymentsRule : ConsensusRule
     {
         /// <inheritdoc />
         /// <exception cref="ConsensusErrors.InvalidPrevTip">The tip is invalid because a reorg has been detected.</exception>
```

Comment:
from line 20 - it probably does not belong here right?

Reply:
yeah there is an Obsoulete property to remove after CM activates
I guess we can just break current consensus and have the node in a not secure state for a few days while we activate CM